modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -229,30 +229,30 @@ and _options_ is a comma-separated list of bonding options. Enter `modinfo bondi
 
 * When you create a bonded interface using `bond=`, you must specify how the IP address is assigned and other
 information for the bonded interface.
-
++
 ** To configure the bonded interface to use DHCP, set the bond's IP address to `dhcp`. For example:
 +
 [source,terminal]
 ----
-bond=bond0:em1,em2:mode=active-backup
 ip=bond0:dhcp
 ----
-
++
 ** To configure the bonded interface to use a static IP address, enter the specific IP address you want and related information. For example:
 ifndef::ibm-z[]
 +
 [source,terminal]
 ----
-bond=bond0:em1,em2:mode=active-backup
 ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
 ----
 endif::ibm-z[]
 ifdef::ibm-z[]
-
++
 [source,terminal]
 ----
-bond=bond0:em1,em2:mode=active-backup,fail_over_mac=1
-ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
+bond=bond0:enp1s0,enp2s0:mode=active-backup
+ip=bond0:dhcp
+ip=:::::enp1s0:none:00:1A:4A:16:01:5A
+ip=:::::enp2s0:none:00:1A:4A:16:01:5A
 ----
 
 Always set the `fail_over_mac=1` option in active-backup mode, to avoid problems when shared OSA/RoCE cards are used.
```

Comment:
So this is a problem.  This is IBM-Z specific, but it means we have to adjust the other file below, since technically we support `fail_over_mac=1`